src/driver/sqljs/SqljsConnectionOptions.ts

```diff
@@ -0,0 +1,32 @@
+import {BaseConnectionOptions} from "../../connection/BaseConnectionOptions";
+
+/**
+ * Sqlite-specific connection options.
+ */
+export interface SqljsConnectionOptions extends BaseConnectionOptions {
```

Comment:
I would improve all comments in this file - make them more concrete and simple for users understanding